stable/redis/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: redis
-version: 10.4.2
+version: 10.4.3
```

Comment:
According to [semver](http://semver.org/):
> Given a version number `MAJOR.MINOR.PATCH`, increment the:
> - MAJOR version when you make incompatible API changes,
> - MINOR version when you add functionality in a backwards-compatible manner, and
> - PATCH version when you make backwards-compatible bug fixes.

In this case, I think this PR is adding a new functionality instead of fixing a bug, so we can bump the minor version -> `10.5.0`